src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py

```diff
@@ -640,7 +640,7 @@ def __call__(
         width: Optional[int] = None,
         num_inference_steps: int = 28,
         timesteps: List[int] = None,
-        guidance_scale: float = 7.0,
+        guidance_scale: float = 5.0,
```

Comment:
thanks! can you change the comments (doc string) instead? we prefer not to change the default value because it will cause backward breaking

Reply:
Okay ~